go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-cty-funcs v0.0.0-20200930094925-2721b1e36840
 	github.com/hashicorp/hcl/v2 v2.8.2
-	github.com/moby/buildkit v0.12.1-0.20230804094609-b49a8873179b
+	github.com/moby/buildkit v0.12.1-0.20230907220514-cbfd4023383d
```

Comment:
Can you add a comment saying that `BUILDKIT_VERSION` in the Dockerfile should be changed as well when updating this module?

Reply:
Isn't it the same as when updating runc/containerd? The binaries should be updated along with the code, but there are legitimate cases where it may not be necessary - e.g. a change in the buildkit test harnesses, mean no `BUILDKIT_VERSION` bump is necessary.